tensorboard/plugins/scalar/tf_scalar_dashboard/tf-scalar-card.html

```diff
@@ -235,6 +237,22 @@
         },
 
         _logScaleActive: Boolean,
+
+        _tooltipColumns: {
+          type: Array,
+          value: function() {
+            const columns = vz_line_chart2.DEFAULT_TOOLTIP_COLUMNS.slice();
+            const ind = columns.findIndex(c => c.title == 'Name');
+            columns.splice(ind, 1, {
```

Comment:
Instead of having to do this splicing, can we just add `this.multiExperiments` to the statusObject for evaluate(), alongside smoothingEnabled()?

Reply:
I believe that breaks the encapsulation of logic. In `statusObject`, the `status` refers to status of the line chart whereas the `multiExperiment`-ness is an invention of applicational component. Line-chart is in no business of knowing about the application (which can make component less shareable).

Reply:
Well, I was thinking it would be plumbed through as user-determined metadata like setSeriesMetadata() itself, but fair enough.  This is fine.